avocado/core/sysinfo.py

```diff
@@ -213,6 +214,45 @@ def stop(self):
         return retcode
 
 
+class JournalctlWatcher(Collectible):
+
+    """
+    Track the content of systemd journal into a compressed file.
+
+    :param logf: Basename of the file where output is logged (optional).
+    """
+
+    def __init__(self, logf=None):
+        if not logf:
+            logf = 'journalctl.gz'
+
```

Comment:
And again previous art on `sysinfo` makes my comment at least weird.  `LogWatcher` also adds this empty line... And others don't.  Small little style issues that my raise big red signs on my eyes... sigh.